app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -289,26 +295,14 @@ protected View onCreateViewHelper(LayoutInflater inflater,
 
         View root = inflater.inflate(fragmentResource, container, false);
 
-        listView = (DragSortListView) root.findViewById(android.R.id.list);
-        txtvEmpty = (TextView) root.findViewById(android.R.id.empty);
-        progLoading = (ProgressBar) root.findViewById(R.id.progLoading);
+        listView = (RecyclerView) root.findViewById(android.R.id.list);
+        layoutManager = new LinearLayoutManager(getActivity());
+        listView.setLayoutManager(layoutManager);
```

Comment:
```
listView.setHasFixedSize(true);
```
listView → recyclerView?

Also, you can take https://github.com/mfietz/AntennaPod/blob/recyclerview/app/src/main/java/de/danoeh/antennapod/view/DividerItemDecoration.java and
```
RecyclerView.ItemDecoration itemDecoration = new DividerItemDecoration(getActivity(), null);
        listView.addItemDecoration(itemDecoration);
```